examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -4978,8 +4981,10 @@ class GeneralCommissioningSetFabric : public ModelCommand
 
         chip::Controller::GeneralCommissioningCluster cluster;
         cluster.Associate(device, endpointId);
-        return cluster.SetFabric(onSuccessCallback->Cancel(), onFailureCallback->Cancel(), reinterpret_cast<char *>(mFabricId),
-                                 reinterpret_cast<char *>(mFabricSecret), mBreadcrumb, mTimeoutMs);
+        return cluster.SetFabric(onSuccessCallback->Cancel(), onFailureCallback->Cancel(),
+                                 chip::ByteSpan(reinterpret_cast<uint8_t *>(mFabricId), strlen(mFabricId)),
```

Comment:
```suggestion
                                 chip::ByteSpan(chip::Uint8::from_char(mFabricId), strlen(mFabricId)),
```
and include CHIPSafeCasts.h